src/net.cpp

```diff
@@ -1512,16 +1512,10 @@ void CConnman::SocketHandler()
             }
         }
 
-        //
-        // Send
-        //
-        if (sendSet)
-        {
-            LOCK(pnode->cs_vSend);
-            size_t nBytes = SocketSendData(pnode);
-            if (nBytes) {
-                RecordBytesSent(nBytes);
-            }
+        if (sendSet) {
+            // Send data
+            size_t bytes_sent = WITH_LOCK(pnode->cs_vSend, {return SocketSendData(pnode);});
```

Comment:
stylistic yocto-nit: could drop the curly braces for the lambda argument here -- according to `git grep WITH_LOCK.*return` on all other places the code is directly passed, w/o putting in a block.

```suggestion
            size_t bytes_sent = WITH_LOCK(pnode->cs_vSend, return SocketSendData(pnode));
```

OTOH maybe the way in this PR is preferred and the others should be adapted (though not in this PR, obviously), IDK.

Reply:
Fixed. Much nicer, thanks!